AmazingMeSample/app/src/main/java/amazingme/app/EnumeratedActivity.java

```diff
@@ -5,13 +5,15 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import amazingme.activities.app.RegisterActivity;
 import amazingme.activities.games.ThreeTouchGame;
 import amazingme.model.AmazingMeAppCompatActivity;
 
 //TODO -> clean this up... surely there is a way to format it better
 public enum EnumeratedActivity {
     //LOGIN,
```

Comment:
can we fill login as well?

nice job keeping this updated, though.

Reply:
ok sure, thanks for the catch.